linera-views/tests/map_view_tests.rs

```diff
@@ -91,10 +91,23 @@ async fn map_view_mutability_check() {
                 let part_key_values = view.map.key_values_by_prefix(vec![u]).await.unwrap();
                 assert_eq!(part_state_vec, part_key_values);
             }
+            for key in &all_keys {
+                let test_map = new_state_map.contains_key(key);
+                let test_view = view.map.get(key).await.unwrap().is_some();
+                assert_eq!(test_map, test_view);
+            }
         }
         if save {
             state_map = new_state_map.clone();
             view.save().await.unwrap();
         }
     }
 }
+
+#[tokio::test]
+async fn map_view_mutability_iter() {
```

Comment:
```suggestion
async fn map_view_mutability_loop() {
```

Or maybe call this one `map_view_mutability` and the other `run_map_view_mutability`?

Reply:
Ok, I prefer your second solution.